geomesa-hbase/geomesa-hbase-datastore/src/main/java/org/locationtech/geomesa/hbase/filters/JSimpleFeatureFilter.java

```diff
@@ -8,32 +8,57 @@
 
 package org.locationtech.geomesa.hbase.filters;
 
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
```

Comment:
yeah, we were trying to lessen our dependency on guava since there are so many versions out there, but it's a work-in-progress so probably not a big deal.